win32ss/user/user32/controls/icontitle.c

```diff
@@ -37,34 +41,6 @@ const struct builtin_class_descr ICONTITLE_builtin_class =
     0                 /* brush */
 };
 
-
-
-#ifndef __REACTOS__
-/***********************************************************************
- *           ICONTITLE_Create
- */
-HWND ICONTITLE_Create( HWND owner )
```

Comment:
If I recall correctly, this "icontitle" is the way minimized (ie. "iconized") windows were shown on Windows 3.x (and NT 3.x)

Reply:
Compromise: let’s move this to it’s own file then since these files NEED to stay as 1:1 with wine changes as possible